package.json

```diff
@@ -38,7 +38,8 @@
   "lint-staged": {
     "*.{js,jsx,ts,tsx,json,sol}": [
       "prettier --write",
-      "eslint --fix"
+      "eslint --fix",
+      "cspell"
```

Comment:
So just to clarify - I understood that you needed to use `cspell **` based on your previous comment? I presume that means that for this usecase it is correct to only use `cspell` here? I don't intend to drag out this review process further but I am curious to learn!

Reply:
I was wrong. I don’t know why, but i wrongly got it into my mind that it would not run. A list of files is being passed to cspell during the lint staged process. Perhaps i had ignored them so they were not triggering, perhaps i was editing the wrong file. I am not sure.